src/init.cpp

```diff
@@ -1408,9 +1408,14 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
                                  fReindexChainState,
                                  nBlockTreeDBCache,
                                  nCoinDBCache,
-                                 nCoinCacheUsage,
-                                 args.GetIntArg("-checkblocks", DEFAULT_CHECKBLOCKS),
-                                 args.GetIntArg("-checklevel", DEFAULT_CHECKLEVEL));
+                                 nCoinCacheUsage);
+        auto rv2 = VerifyLoadedChainstate(chainman,
```

Comment:
In commit "Split off VerifyLoadedChainstate" (019da3dc8493054b456222ef23761dd95130b6a7)

Seems like this only be called if `rv` is `nullopt`, and it should be skipped if `LoadChainstate` fails or is interrupted for shutdown. Otherwise this is running verification code that did not run previously.

Would be better to preserve behavior I think. Either way it would also be good to say in commit description whether behavior is changing or not.